wire-library/buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -0,0 +1,114 @@
+// If false - JS targets will not be configured in multiplatform projects.
```

Comment:
I haven’t yet deleted the other `dependencies.gradle` file. I will do that once the enclosing project is converted to KTS.